### ISSUE TITLE
Settings icon in notification message is not keyboard-accessible

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#57997]
This accessibility bug was filed on the Live Share team; however, it is really an editor issue. I've pasted the bug description below, editing it slightly to remove irrelevant Live Share-specific content.

Repro Steps:
1) Show any notification with the vscode.window.showNotificationMessage API
2) Check whether all the controls in the notification are accessible through keyboard

Actual Result:
"Settings" icon of the notification pop-up is not accessible through keyboard
NOTE: Same is the issue with all the notifications that pop-up in the application

Expected Result:
All the controls in the notification pop-up should be accessible through keyboard

User Impact: 
Keyboard only users will not be able to access the controls present in the notification

Note: Similar issue is observed in Mac as well.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

OS: Windows 10 Enterprise 17134.228

**Comment by bpasero** [comment]
I can tab to the settings icon but I cannot trigger it for some reason.

**Comment by bpasero** [comment]
I pushed a change so that `Space` and `Enter` key show the menu when focus is on the configuration action.

**Comment by cait1inhennessy** [comment]
@bpasero Thank you!

**Comment by cait1inhennessy** [comment]
@bpasero I can tab to the settings icon but hitting `Enter` or `Space` doesn't make the "manage extension" box pop out.  Am I missing something?  I'm using the latest version of insiders.

**Comment by bpasero** [comment]
Something is weird...

**Comment by bpasero** [comment]
This seems to be an issue with the custom context menu actually as it works with the native one:

![image](https://user-images.githubusercontent.com/900690/46189070-129b3780-c2ed-11e8-9249-ce1a585434f8.png)

**Comment by joaomoreno** [comment]
It's an issue in the dropdown widget.
